modules/installation-azure-service-principal.adoc

```diff
@@ -184,7 +181,8 @@ $ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
 .Example output
 [source,terminal]
 ----
-Changing "<service_principal>" to a valid URI of "http://<service_principal>", which is the required format used for service principal names
+Changing "<service_principal>" to a valid URI of "http://<service_principal>",
+which is the required format used for service principal names
 Retrying role assignment creation: 1/36
 Retrying role assignment creation: 2/36
 Retrying role assignment creation: 3/36
```

Comment:
@MayXuQQ 
Line 206: Does the SP need both `Contributor` and `User Access Administrator` roles? Or just  `User Access Administrator`?

It looks like the AD Graph permission can be removed, as well as the reference to the CCO documentation, since it no longer mentions Azure SP requiring additional permissions.

Reply:
$ az ad sp create-for-rbac --role Contributor --name <service_principal> --scopes ....
$az role assignment create --role "User Access Administrator" \
  --assignee-object-id $(az ad sp show --id <sp id> --query id -o tsv)